auth0_flutter/lib/src/web/js_interop.dart

```diff
@@ -106,7 +106,7 @@ class GetTokenSilentlyAuthParams {
 
 @JS()
 @anonymous
-class GetTokenSilentlyOptions {
+extension type GetTokenSilentlyOptions._(JSObject _) implements JSObject {
   external GetTokenSilentlyAuthParams? get authorizationParams;
   external String? get cacheMode;
   external num? get timeoutInSeconds;
```

Comment:
This should also be a `JSNumber?` ? Maybe we should do a pass for the remaining `num` declarations.